wire-runtime/src/main/java/com/squareup/wire/RuntimeEnumAdapter.java

```diff
@@ -24,14 +24,20 @@
  */
 final class RuntimeEnumAdapter<E extends WireEnum> extends ProtoAdapter<E> {
   private final Class<E> type;
-  private final Method fromValueMethod;
+  private Method fromValueMethod; // Loaded lazily to avoid reflection during class loading.
```

Comment:
`volatile` ? Probably doesn't heart much if we retrieve the method twice though.